context/grpc/src/main/java/brave/context/grpc/GrpcCurrentTraceContext.java

```diff
@@ -0,0 +1,26 @@
+package brave.context.grpc;
+
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.TraceContext;
+import io.grpc.Context;
+
+/** Manages scope with a {@link Context} as opposed to a thread local. */
+public final class GrpcCurrentTraceContext extends CurrentTraceContext {
```

Comment:
I wrapped the log4j2 context and that didn't work. But just to make sure I'm accessing it correctly `ThreadContext.get("traceId")` is the expected usage?